Tests/NatsTests/Unit/HeadersTests.swift

```diff
@@ -48,10 +52,12 @@ class HeadersTests: XCTestCase {
         hm.append(try! NatsHeaderName("foo"), NatsHeaderValue("baz"))
         hm.insert(try! NatsHeaderName("bar"), NatsHeaderValue("foo"))
 
-        let expected = "NATS/1.0\r\nfoo:bar\r\nfoo:baz\r\nbar:foo\r\n\r\n"
-        let byteArray: [UInt8] = Array(expected.utf8)
+        let expected = [
+            "NATS/1.0\r\nfoo:bar\r\nfoo:baz\r\nbar:foo\r\n\r\n",
+            "NATS/1.0\r\nbar:foo\r\nfoo:bar\r\nfoo:baz\r\n\r\n",
+        ]
 
-        XCTAssertEqual(hm.toBytes(), byteArray)
+        XCTAssertTrue(expected.contains(String(bytes: hm.toBytes(), encoding: .utf8)!))
```

Comment:
this one was tripping me up because order was different for me. Gut I haven't checked lately, and it's just me, so no worries.